fineract-provider/src/main/resources/db/changelog/tenant/parts/0014_remove_unused_jobs.xml

```diff
@@ -21,6 +21,9 @@
 -->
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
     <changeSet author="fineract" id="1">
+        <sql>
```

Comment:
Blocker change!
Please revert this.

I believe we should never remove history entries!

Also, please never modify an already existing and committed changeset!
Reason: it will fail for everyone who already pulled that changeset

Reply:
@adamsaghy 
OK! I can revert without problem, but It'll fails for everyone that wants to run this with run_job_history data, that was the reason that I updated this, the next screen shows the error

<img width="1565" alt="Screen Shot 2022-06-02 at 21 34 39" src="https://user-images.githubusercontent.com/44206706/171862306-68b6e394-d04e-442d-8522-52afef142e40.png">

Reply:
@taskain7 can you please take a look at the above?

Reply:
@josehernandezfintecheandomx in this case, you are right. This is a must. However pls add a "validChecksum" as well, that way it will not fail for others (who might already executed this changelog)

Reply:
@josehernandezfintecheandomx in this case, you are right. This is a must. However pls add a "validChecksum" as well, that way it will not fail for others (who might already executed this changelog)

Reply:
nice catch @josehernandezfintecheandomx , I didn't find this issue when I made this change.
IMHO we can keep this delete in this PR, just make sure that the migration can run for anybody (who already ran my changes and aalso who didn't)

Reply:
Resolved using validChecksum as suggested Adam

Reply:
@josehernandezfintecheandomx In the meantime this problem was solved by an another strategy. The FK was removed from the job_run_history table, so we dont need to delete the history entry of the "deprecated" jobs.


Would you be so kind to remove this whole changeset from this PR?

Reply:
@josehernandezfintecheandomx In the meantime this problem was solved by an another strategy. The FK was removed from the job_run_history table, so we dont need to delete the history entry of the "deprecated" jobs.


Would you be so kind to remove this whole changeset from this PR?